packages/playground/src/calculator/pricing_calculator.vue

```diff
@@ -0,0 +1,295 @@
+                </InputTooltip>
+              </VCol>
+
+              <VCol cols="4">
+                <InputTooltip
+                  inline
+                  tooltip=" A certified node will receive 25% more reward compared to a non-certified node."
+                >
+                  <VSwitch color="primary" inset label="Certified Node" hide-details v-model="resources.certified" />
+                </InputTooltip>
+              </VCol>
+
+              <VCol cols="4">
+                <InputTooltip
+                  inline
+                  tooltip="An Internet Protocol version 4 address that is globally unique and accessible over the internet"
+                >
+                  <VSwitch color="primary" inset label="With a Public IPv4" hide-details v-model="resources.ipv4" />
+                </InputTooltip>
+              </VCol>
+
+              <VCol
+                cols="4"
+                v-if="userBalance"
+                @vue:unmounted="
+                  resources.useCurrentBalance = true;
+                  priceTask.run();
+                "
+              >
+                <input-tooltip inline tooltip="Use current balance to calculate the discount.">
+                  <VSwitch
+                    color="primary"
+                    inset
+                    label="Use current balance"
+                    hide-details
+                    v-model="resources.useCurrentBalance"
+                  />
+                </input-tooltip>
+              </VCol>
+            </VRow>
+          </VForm>
+
+          <VDivider class="my-8" />
+          <VRow v-if="valid && priceTask.error">
+            <VCol cols="12">
+              <VAlert type="error" :text="normalizeError(priceTask.error, 'Failed to calculate price.')" />
+            </VCol>
+          </VRow>
+
+          <VRow class="text-center text-body-1 text-black" v-else-if="valid">
+            <VCol cols="6">
+              <div
+                class="rounded pa-4 h-100 d-flex justify-center align-center"
+                :style="{ background: computePackageColor(priceTask.data?.dedicatedPackage.package) }"
+              >
+                <p>
+                  Cost of reserving a <strong v-text="'Dedicated Node'" /> of the same specifications<br />
+                  <strong v-text="'Loading...'" v-if="priceTask.loading" />
+                  <strong v-else v-text="dedicatedPriceUSD + ' USD/month, ' + dedicatedPriceTFT + ' TFT/month'" />. A
+                  user can reserve an entire node then use it exclusively to deploy solutions
+                </p>
+              </div>
+            </VCol>
+            <VCol cols="6">
+              <div
+                class="rounded pa-4 h-100 d-flex justify-center align-center"
+                :style="{ background: computePackageColor(priceTask.data?.sharedPackage.package) }"
+              >
+                <p>
+                  Cost of reservation on a <strong v-text="'Shared Node'" /><br />
```

Comment:
I'm not sure if we should mention the package's name here or if the background color is enough?

Reply:
@maayarosama what do u mean? 
please add some ss if something is not working correctly

Reply:
Let's ignore my suggestion here as it's not needed